twitter_bot.py

```diff
@@ -82,9 +82,11 @@ def isSafeKeyword(self, tweetText):
 
     def nawab_get_id(self):
         ### Read the last retweeted id from a file
-        tid = pd.read_csv(self.dirpath + 'tid_store.csv')
-        last = tid['Id'].iloc[-1]
-        return last
+        id_list = []
+        tid_store = pd.read_csv(self.dirpath + 'tid_store.csv') 
+        for index, tid in tid_store.iterrows():
+           id_list.append(tid['Id'])
+        return id_list 
```

Comment:
Do you have to convert it to a list in the core code itself? Isn't it wiser to collect the tid at the test and covert it to a list there?